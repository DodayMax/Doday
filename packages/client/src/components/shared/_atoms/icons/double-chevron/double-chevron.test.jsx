import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('DoubleChevronIcon', () => {
  it('renders correctly without props', () => {
    const icon = shallow(<Icons.DoubleChevronIcon />);

    expect(icon).toMatchSnapshot();
  });
});
