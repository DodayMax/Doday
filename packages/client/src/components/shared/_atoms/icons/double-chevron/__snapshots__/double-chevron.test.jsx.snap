// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoubleChevronIcon renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <path
        d="M9.991 20C15.504 20 20 15.521 20 9.991S15.521 0 9.991 0 0 4.479 0 9.991C0 15.504 4.479 20 9.991 20zm0-18.605c4.738 0 8.596 3.86 8.596 8.596 0 4.738-3.858 8.596-8.596 8.596-4.737 0-8.596-3.858-8.596-8.596 0-4.737 3.86-8.596 8.596-8.596zm.948 13.592a.69.69 0 0 0 .5.207.707.707 0 0 0 .5-1.206L7.888 9.94l4.049-4.049a.707.707 0 0 0-1-.999L6.392 9.44a.69.69 0 0 0-.207.5c0 .19.07.361.207.5l4.548 4.547z"
        fill="#494949"
        fillRule="evenodd"
      />,
      "className": "icon ",
      "height": 20,
      "viewBox": "0 0 20 20",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "d": "M9.991 20C15.504 20 20 15.521 20 9.991S15.521 0 9.991 0 0 4.479 0 9.991C0 15.504 4.479 20 9.991 20zm0-18.605c4.738 0 8.596 3.86 8.596 8.596 0 4.738-3.858 8.596-8.596 8.596-4.737 0-8.596-3.858-8.596-8.596 0-4.737 3.86-8.596 8.596-8.596zm.948 13.592a.69.69 0 0 0 .5.207.707.707 0 0 0 .5-1.206L7.888 9.94l4.049-4.049a.707.707 0 0 0-1-.999L6.392 9.44a.69.69 0 0 0-.207.5c0 .19.07.361.207.5l4.548 4.547z",
        "fill": "#494949",
        "fillRule": "evenodd",
      },
      "ref": null,
      "rendered": null,
      "type": "path",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <path
          d="M9.991 20C15.504 20 20 15.521 20 9.991S15.521 0 9.991 0 0 4.479 0 9.991C0 15.504 4.479 20 9.991 20zm0-18.605c4.738 0 8.596 3.86 8.596 8.596 0 4.738-3.858 8.596-8.596 8.596-4.737 0-8.596-3.858-8.596-8.596 0-4.737 3.86-8.596 8.596-8.596zm.948 13.592a.69.69 0 0 0 .5.207.707.707 0 0 0 .5-1.206L7.888 9.94l4.049-4.049a.707.707 0 0 0-1-.999L6.392 9.44a.69.69 0 0 0-.207.5c0 .19.07.361.207.5l4.548 4.547z"
          fill="#494949"
          fillRule="evenodd"
        />,
        "className": "icon ",
        "height": 20,
        "viewBox": "0 0 20 20",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M9.991 20C15.504 20 20 15.521 20 9.991S15.521 0 9.991 0 0 4.479 0 9.991C0 15.504 4.479 20 9.991 20zm0-18.605c4.738 0 8.596 3.86 8.596 8.596 0 4.738-3.858 8.596-8.596 8.596-4.737 0-8.596-3.858-8.596-8.596 0-4.737 3.86-8.596 8.596-8.596zm.948 13.592a.69.69 0 0 0 .5.207.707.707 0 0 0 .5-1.206L7.888 9.94l4.049-4.049a.707.707 0 0 0-1-.999L6.392 9.44a.69.69 0 0 0-.207.5c0 .19.07.361.207.5l4.548 4.547z",
          "fill": "#494949",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
